resources/containers/settings-button.tsx

```diff
@@ -14,7 +14,7 @@ const SettingsButton = () => {
         width={13}
         height={13}
       />
-      <span>{t("settings")}</span>
+      <span>{t("Settings")}</span>
```

Comment:
Setting buttons is a `bottom-nav.button` or `highlight-nav.button` 

We can pass the label as a prop. 
Otherwise is not rehuable any more :D. (resources are rehusable by nature)